Bottom: 0193ea3f86e40817c7a07fa255410a2807ce8df3
Top:    ce7ef4114d141880b0e48f6f365c5c2c0dfd730c
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-12-15 17:58:43 -0800

Refresh of udpong

---

diff --git a/examples/udpong.c b/examples/udpong.c
index af8deb9..d7d6962 100644
--- a/examples/udpong.c
+++ b/examples/udpong.c
@@ -132,7 +132,7 @@ static void init_latency_test(int size)
 	size_str(sstr, sizeof sstr, size);
 	snprintf(test_name, sizeof test_name, "%s_lat", sstr);
 	transfer_size = size;
-	transfer_count = size_to_count(transfer_size) / 10;
+	transfer_count = 10;//size_to_count(transfer_size) / 10;
 	echo = 1;
 }
 
@@ -143,7 +143,7 @@ static void init_bandwidth_test(int size)
 	size_str(sstr, sizeof sstr, size);
 	snprintf(test_name, sizeof test_name, "%s_bw", sstr);
 	transfer_size = size;
-	transfer_count = size_to_count(transfer_size);
+	transfer_count = 10;//size_to_count(transfer_size);
 	echo = 0;
 }
